decidim-admin/app/views/decidim/admin/newsletters/select_recipients_to_deliver.html.erb

```diff
@@ -60,7 +60,8 @@
             </div>
             <div class="card-section">
               <div class="row column">
-                <%= scopes_picker_filter f, :scope_ids, help_text: t(".scopes_help") %>
+                <%= scopes_select_field f, :scope_ids %>
```

Comment:
We could replace the entire block from line: 56 to 68 with the following snippet: 
This will: 
- convert the select box to a multiple selection
- make the select box the same size as the one in participatory process 
- align the scopes box with the rest

![image](https://github.com/user-attachments/assets/21432ed9-a692-4332-a9ea-60a9313d3904)


```
        <% if current_user.admin? %>
          <div class="card mb-2">
            <div class="card-divider">
              <h2 class="card-title"><%= t ".select_scopes" %></h2>
            </div>
            <div class="card-section">
              <div class="cell small-12 medium-6">
                <%= f.select(
                      :scope_ids,
                      ordered_scopes_descendants_for_select(false),
                      { include_blank: false },
                      { class: "chosen-select", multiple: true, size: 5 }
                    ) %>
                <span class="help-text"> <%= t(".scopes_help") %> </span>
              </div>
            </div>
          </div>
        <% end %>
```

The other refactor you can do, is to remove the `f.select ... ` from above snippet, and replace it with a : 
`<%= scopes_select_field f, :scope_ids, { include_blank: false }, { class: "chosen-select", multiple: true, size: 5 } %>`

Then patch: `decidim-core/app/helpers/decidim/scopes_helper.rb`  like follows: 

```
    def scopes_select_field(form, name, root: false, options: {}, html_options: {})
      options = options.merge(include_blank: I18n.t("decidim.scopes.prompt")) unless options.has_key?(:include_blank)

      form.select(
        name,
        ordered_scopes_descendants_for_select(root),
        options, 
        html_options
      )
    end
```

Reply:
@alecslupu the second part to your patch (replacing the *f.select*), delivers a Rails error for some reason, stating the no. arguments used in the method.

Reply:
can you be more explicit ?